packages/components/app/styles/components/breadcrumb.scss

```diff
@@ -125,22 +121,28 @@ $hds-breadcrumb-item-visual-horizontal-padding: 4px;
 .hds-breadcrumb__text {
     font-family: var(--token-typography-font-stack-text);
     font-size: 0.8125rem; // 13px
-    line-height: $hds-breadcrumb-item-height; // we want to match the height of the item
+    line-height: 1rem; // 16px
     overflow: hidden;
+    // we use the extra vertical padding to force the height of the parent item to be exactly $hds-breadcrumb-item-height
+    padding: calc((#{$hds-breadcrumb-item-height} - 1rem) / 2) 0;
     text-decoration: underline;
     text-decoration-color: transparent;
     text-overflow: ellipsis;
     white-space: nowrap;
```

Comment:
I am not going to block on this because I don't have an alternative solution to offer/suggest just yet. But this will be an accessibility issue- there is no way for a keyboard-only user to access truncated content. 

![image](https://user-images.githubusercontent.com/4587451/158390233-4a435709-86d1-4435-a6fe-99322cb291b1.png)

Reply:
OK.

Reply:
@MelSumner added as "known issue" in https://github.com/hashicorp/design-system/pull/71/commits/fa26a4b3e4684fc9fb8a670d6ad4a8c90f0e2db5